src/base/Algorithms.hh

```diff
@@ -19,14 +20,14 @@ namespace celeritas
 //! Implement perfect forwarding with device-friendly functions.
 template<class T>
 CELER_CONSTEXPR_FUNCTION T&&
-forward(typename std::remove_reference<T>::type& v) noexcept
+cforward(typename std::remove_reference<T>::type& v) noexcept
```

Comment:
For the record, could you augment the git log to explain why/how this solution is superior to 'simply' requiring to always use the fully qualified name (`celeritas::forward`).

Reply:
Will do. The problem is essentially that algorithms defined in the `std` namespace use unqualified names (`swap`) to enable argument-dependent lookup to find custom `swap` methods.

We might be able to get away with `forward` if we always qualify it with `celeritas::`, but I think I hit issues in testing when calling our algorithms on `std` containers/iterators.

Reply:
Changed as noted below.